cmd/notation/verify.go

```diff
@@ -161,12 +172,31 @@ func resolveReference(ctx context.Context, opts *SecureFlagOpts, reference strin
 	return ref, nil
 }
 
-func printMetadataIfPresent(outcome *notation.VerificationOutcome) {
+func printResult(outputFormat, reference string, outcome *notation.VerificationOutcome) error {
+	output := verifyOutput{Reference: reference, Result: "Success", UserMetadata: map[string]string{}}
```

Comment:
nit: usually its best practice to start with failure mode and add then override data, unless you have all the data you need to call success.